### DISCUSSION TITLE
Need an example of how to use Vuex in Nuxt3 project

### DISCUSSION THREAD

**Discussion by saifobeidat** [nuxt/nuxt#18070] [General]
Hello Everyone,
Does anyone have a sample project of how to configure Vuex with Nuxt3 project? And how to use the store in pages/components

Thanks

**Comment by aligzl** [comment]
use pinia. it's more lightweight and easy to use [link](https://pinia.vuejs.org/)

**Reply by saifobeidat** [reply]
Thanks @aligzl I heard about it, unfortunately it's hard to use it because we are migrating a vuejs project that uses Vuex to Nuxt3.. we don't need to do major changes

**Comment by wobsoriano** [comment]
Not supported yet https://v3.nuxtjs.org/getting-started/migration/

**Comment by vedmant** [comment]
You can try a module https://github.com/vedmant/nuxt3-vuex it should support store folder and nested modules, but may not support separated actions/mutations etc files.
